docs/how-to/amplication-ai-examples.md

```diff
@@ -8,7 +8,7 @@ pagination_prev: how-to/amplication-ai
 
 # Amplication AI Examples
```

Comment:
@dericksozo - General comments- the examples here are great, but they are "simple" in a way that they just mimic human actions that can be done easily in our platform. Jovu can assist also in actions like suggesting the required entities, suggesting services, fields etc. So the user can just ask to create a blog project, and Jovu can assist with suggesting the required services, entities etc.
I will send you a video that can demonstrate the above